src/libcore/ops.rs

```diff
@@ -2649,6 +2649,35 @@ mod impls {
 
 /// Trait that indicates that this is a pointer or a wrapper for one,
 /// where unsizing can be performed on the pointee.
+///
+/// See the [DST coercion RfC][dst-coerce] and [the nomicon entry on coercion][nomicon-coerce]
+/// for more details.
+///
+/// For builtin types, pointers to `T` will coerce to pointers to `U` if `T: Unsize<U>`
```

Comment:
"For builtin types" sounds unclear; perhaps reword this to say something to the effect of "for builtin pointer types".